src/eureka/S3_data_reduction/plots_s3.py

```diff
@@ -426,3 +427,79 @@ def driftywidth(data, meta):
     plt.savefig(meta.outputdir+fname, bbox_inches='tight', dpi=300)
     if not meta.hide_plots:
         plt.pause(0.2)
+
+
+def residualBackground(data, meta, m):
+    '''Plot the median, BG-subtracted frame to study the residual BG region and
+    aperture/BG sizes. (Fig 3304)
+
+    Parameters
+    ----------
+    data : Xarray Dataset
+        The Dataset object.
+    meta : eureka.lib.readECF.MetaClass
+        The metadata object.
+    m : int
+        The file number.
+
+    Returns
+    -------
+    None
+
+    Notes
+    -----
+    History:
+
+    - 2022-07-29 KBS
+        First version
+    '''
+    # Median flux of segment
+    flux = np.median(data.flux, axis=0)
```

Comment:
This should probably be a masked array using data.mask and using np.ma calls throughout